ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/IndividualMapper.java

```diff
@@ -33,9 +33,12 @@ public CCDIndividual to(Individual individual) {
         individual.getRepresentative()
             .ifPresent(representative -> builder.representative(representativeMapper.to(representative)));
 
+        if (individual.getDateOfBirth() != null) {
+            builder.dateOfBirth(individual.getDateOfBirth().format(DateTimeFormatter.ISO_DATE));
+        }
+
         return builder
             .name(individual.getName())
-            .dateOfBirth(individual.getDateOfBirth().format(DateTimeFormatter.ISO_DATE))
```

Comment:
will it not fail again?

Reply:
removed here and doing null check ad adding